src/libraries/Microsoft.PowerFx.Core/Syntax/Nodes/ReplaceableNode.cs

```diff
@@ -9,9 +9,9 @@
 
 namespace Microsoft.PowerFx.Core.Syntax.Nodes
 {
-    internal sealed class ReplaceableNode : TexlNode
+    public sealed class ReplaceableNode : TexlNode
     {
-        public ReplaceableNode(ref int idNext, ReplaceableToken tok)
```

Comment:
>ReplaceableNode [](http://example.com/codeflow?start=15&length=15)

This node type was for a feature that never shipped and should be cleaned up.

Reply:
Are you saying we should we remove this class altogether?

Reply:
Yes, although that can be done independently before GA if this is checked in.

Reply:
Resolved by #273